charts/webapi/templates/deployment.yaml

```diff
@@ -77,6 +77,42 @@ spec:
               value: "Host=$(DB_HOST);Port=5432;Database=$(POSTGRESQL_DATABASE);Username=$(POSTGRESQL_USER);Password=$(POSTGRESQL_PASSWORD)"
             - name: ApplicationUrl
               value: "https://{{ if $isProd }}{{else}}{{ $release }}.{{end}}{{ $domain }}"
+            - name: BCProviderClient__ClientID
+              {{- if $isPR }}
+              value: dev-bc-provider # Use dev secret for pr
+              {{ else }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ ternary $ocEnv $release $isPR }}-bc-provider
+                  key: BCProviderClient__ClientID
+              {{ end }}
+            - name: BCProviderClient__TenantID
+              {{- if $isPR }}
+              value: dev-bc-provider # Use dev secret for pr
+              {{ else }}
+              valueFrom:
+                secretKeyRef: 
+                  name: {{ ternary $ocEnv $release $isPR }}-bc-provider
+                  key: BCProviderClient__TenantID
+              {{ end }}
+            - name: BCProviderClient__ClientSecret
+              {{- if $isPR }}
+              value: dev-bc-provider # Use dev secret for pr
+              {{ else }}
+              valueFrom:
+                secretKeyRef: 
+                  name: {{ ternary $ocEnv $release $isPR }}-bc-provider
+                  key: BCProviderClient__ClientSecret
+              {{ end }}
+            - name: BCProviderClient__Domain
+              {{- if $isPR }}
+              value: dev-bc-provider # Use dev secret for pr
+              {{ else }}
+              valueFrom:
+                secretKeyRef: 
+                  name: {{ ternary $ocEnv $release $isPR }}-bc-provider
+                  key: BCProviderClient__Domain
+              {{ end }}
```

Comment:
Wouldn't we be able to add the secrets by just adding one line to 
```yaml
envFrom:
  - secretRef:
    name: bc-provider
  - secretRef:
    name: keycloak-secrets
```

Reply:
Yea should be good to do that. I've adjusted the PR to do this.